### PR TITLE
Final layout changed

### PR DESCRIPTION
**Pull request by sophiabulcock**
Fixed layout changes

### CODE REVIEW

**Comment by mqharris** [comment]
The directory structure has changed since you last pulled

There is no longer a `app_layout_1.R` file and there is no longer a `src` dir

**Comment by mqharris** [comment]
Also please give branches descriptive names, your future self will thank you 🙃

File: app_layout_1.R [hunk 1]
```diff
@@ -136,7 +144,7 @@ published between 1950 and 2021."))))}
 first_card = dbcCard(
 dbcCardBody(
 htmlDiv(id="left-column",
-className="four columns",list(description_card(),htmlBr(),generate_control_card()))))
+className="four columns",list(description_card(),htmlBr(),generate_control_card(), htmlBr(), htmlBr(), data_set_description(), htmlBr(),htmlBr(),htmlBr()))))
```
Comment (WraySmith) [inline app_layout_1.R:148]: that line of code should be split

File: app_layout_1.R [hunk 2]
```diff
@@ -301,10 +313,14 @@ tab_selected_style = list(
...
 # layout made up of cards
-app$layout(htmlDiv(list(dbcContainer(htmlDiv(id="title_top",className="title on top", list(title()))), htmlDiv(dccTabs(id="tabs_in_app", value="tab-1"
-,list(dccTab(label="Game Dynamics Over Time", value='tab-1',children=list(dbcRow(list(dbcCol(first_card, width=3), dbcCol(second_card, width=9))), dbcRow(list(dbcCol(third_card, width=3),dbcCol(list(fourth_card,fifth_card), width=9))), dbcRow(dbcCol(ninth_card, width=3))), style=tab_style,
-selected_style=tab_selected_style),dccTab(label="Top Games", value="tab-2",children=list(dbcRow(list(dbcCol(sixth_card, width=3), dbcCol(list(seventh_card,eight_card), width=9)))),style=tab_style,
-selected_style=tab_selected_style)))))))
+app$layout(htmlDiv(list(dbcContainer(htmlDiv(id="title_top",className="title on top", list(title()))),
+htmlDiv(dccTabs(id="tabs_in_app", value="tab-1",list(dccTab(label="Game Dynamics Over Time", value='tab-1',
+children=list(dbcRow(list(dbcCol(first_card, width=3), dbcCol(second_card, width=9))),
+dbcRow(list(dbcCol(third_card, width=3),dbcCol(list(fourth_card,fifth_card), width=9)))), 
+style=tab_style,selected_style=tab_selected_style),
```
Comment (WraySmith) [inline app_layout_1.R:320]: recommend splitting this up a bit more/adding comments in, it's still pretty heavy

**Comment by WraySmith** [comment]
added a couple comments on lines that should be cut down a bit as well as @mqharris comments

**Pull request closed without merging**